test/functional/feature_loan.py

```diff
@@ -231,7 +231,9 @@ def run_test(self):
         assert_equal(auctions[1]['auctionBid'], "515.00000000@TSLA")
         assert_equal(auctions[2]['auctionBid'], "555.50000000@TSLA")
 
-        auctionsV = self.nodes[0].listauctionhistory(vaultId1)
+        auctionsV = self.nodes[0].listauctionhistory(vaultId1, {"index": 1})
```

Comment:
related to https://github.com/DeFiCh/ain/pull/1338/files#r899757282,
This query asks for all the subsequent records from the pagination start point AuctionHistoryKey(0,{},{},1)